nni/compression/pytorch/speedup/quantization_speedup/calibrator.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import tensorrt as trt
+import os
+import pycuda.driver as cuda
+from urllib.request import urlretrieve
+import numpy as np
+
+
+class Calibrator(trt.IInt8Calibrator):
+    def __init__(self, training_data, cache_file, batch_size=64, algorithm=trt.CalibrationAlgoType.ENTROPY_CALIBRATION_2):
+        """
+        Parameters
+        ----------
+        training_data : numpy array
+            The data using to calibrate quantization model
+        cache_file : str
+            The path user want to store calibrate cache file
+        batch_size : int
+            The batch_size of calibrating process
+        algorithm : tensorrt.tensorrt.CalibrationAlgoType
```

Comment:
list available algorithms here and add reference

Reply:
Added.